app/onboarding/views/account_activated.py

```diff
@@ -0,0 +1,41 @@
+from app.onboarding.base import onboarding_bp
+from enum import Enum
+from flask import redirect, render_template, request, url_for
+
+
+CHROME_EXTENSION_LINK = "https://chrome.google.com/webstore/detail/simpleloginreceive-send-e/dphilobhebphkdjbpfohgikllaljmgbn"
+FIREFOX_EXTENSION_LINK = "https://addons.mozilla.org/firefox/addon/simplelogin/"
+
+
+class Browser(Enum):
```

Comment:
Can we also add Edge? The store link for Edge is https://microsoftedge.microsoft.com/addons/detail/simpleloginreceive-sen/diacfpipniklenphgljfkmhinphjlfff

Reply:
Added Edge to the browser detection: https://github.com/simple-login/app/pull/1004/commits/f5d0142aba0cde1a577096e8fe70b708f7cab0d6